src/pages/Editor.vue

```diff
@@ -1156,6 +1186,15 @@ export default {
     ]),
     ...mapActions("generic", ["showSharePlioDialog", "showEmbedPlioDialog"]),
     ...Utilities,
+    copyPlioDraftLink() {
+      // copies the link to the clipboard
+      var success = Utilities.copyToClipboard(
```

Comment:
```suggestion
      let success = Utilities.copyToClipboard(
```
No need to `var` here right?